src/main/java/uk/gov/hmcts/cmc/claimstore/events/CCDEventProducer.java

```diff
@@ -91,4 +93,13 @@ public void createCCDResponseDeadlineEvent(
     public void createCCDLinkDefendantEvent(String claimReferenceNumber, String defendantId) {
         publisher.publishEvent(new CCDTestingLinkDefendantEvent(claimReferenceNumber, defendantId));
     }
+
+    public void createCCDRejectOrganisationPaymentPlanEvent(Claim claim, String authorisation) {
+        publisher.publishEvent(new CCDRejectOrganisationPaymentPlanEvent(claim, authorisation));
+
+    }
+
```

Comment:
drop extra line

Reply:
This is single extra line to separate two methods.